setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="pipelineinfrastructure",
```

Comment:
What's the standard naming convention here? Is it 

`pipelineinfrastructure` or `pipeline-infrastructure`

Reply:
As far as I can tell, there's little naming convention on this. Some people use CamelCase, others lower case, either in a single word, or separated by - or _. I used lower case as most python packages that I've used so far seem to be lower case, but I see that CoreDataModules uses upper case as the name, and lowercase for the package. Do you have any preference?